CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+CHANGELOG
+=========
+
```

Comment:
dont forget https://keepachangelog.com/en/1.0.0/
esp the diff links per version can be useful e.g. https://raw.githubusercontent.com/jet/FsCodec/master/CHANGELOG.md